Jenkinsfile

```diff
@@ -170,29 +170,45 @@ pipeline {
                         }
                     }
                     steps {
+                         githubNotify credentialsId: 'daos-jenkins-commit-status',
+                                      description: env.STAGE_NAME,
+                                      context: "build" + "/" + env.STAGE_NAME,
+                                      status: "PENDING"
                         checkoutScm withSubmodules: true
-                        sh '''rm -rf artifacts/
-                              mkdir -p artifacts/
-                              if make srpm; then
-                                  if make mockbuild; then
-                                      (cd /var/lib/mock/epel-7-x86_64/result/ && cp -r . $OLDPWD/artifacts/)
-                                      createrepo artifacts/
-                                  else
-                                      rc=\${PIPESTATUS[0]}
-                                      (cd /var/lib/mock/epel-7-x86_64/result/ && cp -r . $OLDPWD/artifacts/)
-                                      cp -af _topdir/SRPMS artifacts/
-                                      exit \$rc
-                                  fi
-                              else
-                                  exit \${PIPESTATUS[0]}
-                              fi'''
+                        sh label: env.STAGE_NAME,
+                           script: '''rm -rf artifacts/
+                                      mkdir -p artifacts/
+                                      if make srpm; then
+                                          if make mockbuild; then
+                                              (cd /var/lib/mock/epel-7-x86_64/result/ && cp -r . $OLDPWD/artifacts/)
```

Comment:
Not a regression, but why the cd ... && notation when a regular cp should work the same?

Reply:
While i agree that they seem like they should produce the same result, I don't recall specifically if there was a practical difference between the two.  Perhaps just my frame of mind at the time.